src/Spectre.Console/Rendering/Segment.cs

```diff
@@ -79,7 +79,7 @@ public Segment(string text, Style style)
 
         private Segment(string text, Style style, bool lineBreak, bool control)
         {
-            Text = text?.NormalizeNewLines() ?? throw new ArgumentNullException(nameof(text));
+            Text = text.NormalizeNewLines() ?? throw new ArgumentNullException(nameof(text));
```

Comment:
I think we still want the `text?` part since a `text` that is null will throw a null reference exception here.

```suggestion
            Text = text?.NormalizeNewLines() ?? throw new ArgumentNullException(nameof(text));
```

Reply:
text would have had to be defined as `string?` to require a null check here, right?  But that change doesn't belong in this commit regardless. Ripped it out plus a whitespace change that snuck in.

Reply:
@phil-scott-78 Correct, but since Segment is a public API, we can't guarantee that someone doesn't call the method with a null string if nullable reference types are turned off.